docs/docker/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This script requires the local Docker image (playwright:localbuild) by the build.sh
+
+set -ex
+
+CONTAINER_ID="$(docker run --name playwright-docker-test -d -t playwright:localbuild /bin/bash)"
+
+# Reason for using a separate Git repo inside the Docker container instead of the current one is that 'docker cp'
+# Does not respect the .dockerignore or .gitignore
```

Comment:
Would mounting into the docker work?

E.g. the following will mount the current directory as `/home/pwuser/playwright` inside container:

```sh
$ docker run -v $PWD:/home/pwuser/playwright --rm -it playwright:localbuild /bin/bash
```

Reply:
- The reason was for a separate directory that in the normal repo we have `node_modules` which are quite big and `docker cp` does not check if the folder pass the `.dockerignore` file.
- Also not sure if we want to build the project on the host system or in the Docker container. Currently its unfortunately both. Host for building the Docker image and injecting the NPM module and inside for building the project again so lib/ folder is persistent.